### PR TITLE
Update squeezenet and add V1.1 version

### PR DESCRIPTION
**Pull request by Shashi456**
So the default activation in conv layers is identity, while the squeezenet paper mentions a relu activation after the 1st and last convolution.

Also adding the v1.1 version of squeezenet, its 2.4x less computation without sacrificing accuracy. For more check [here](https://github.com/DeepScale/SqueezeNet/tree/master/SqueezeNet_v1.1).

### CODE REVIEW

**Comment by BradLarson** [comment]
Good catch on the missing ReLUs in the model. I missed that in the initial review.

Because this renames SqueezeNet to SqueezeNetV1_0, it will break the newly added inference tests here:     https://github.com/tensorflow/swift-models/blob/master/Tests/ImageClassificationTests/Inference.swift#L95 . Could you also rename the model in the inference test, and maybe add the new SqueezeNetV1_0 as a new test there?

File: Models/ImageClassification/SqueezeNet.swift [hunk 1]
```diff
@@ -114,3 +115,67 @@ public struct SqueezeNet: Layer {
         return convolved2
     }
 }
+
+public struct SqueezeNetV1_1: Layer {
+    public var conv1 = Conv2D<Float>(filterShape: (3, 3, 3, 96), strides: (2, 2), padding: .same,
```
[the remark this answers is not in the document]
Comment (Shashi456) [reply Models/ImageClassification/SqueezeNet.swift]: Yeah, it's 64, i wonder how I missed that.

**Comment by Shashi456** [comment]
@BradLarson done :)

File: Models/ImageClassification/SqueezeNet.swift [hunk 2]
```diff
@@ -115,3 +116,68 @@ public struct SqueezeNet: Layer {
         return convolved2
     }
 }
+
+public struct SqueezeNetV1_1: Layer {
+    public var conv1 = Conv2D<Float>(filterShape: (3, 3, 3, 64), strides: (2, 2), padding: .same,
```
Comment (rxwei) [inline Models/ImageClassification/SqueezeNet.swift]:
```suggestion
    public var conv1 = Conv2D<Float>(
        filterShape: (3, 3, 3, 64), 
        strides: (2, 2), 
        padding: .same,
        activation: relu)
```
Please follow the Google Swift Style Guide or run swift-format.

Comment (Shashi456) [reply Models/ImageClassification/SqueezeNet.swift]: swift-format doesn't modify it this way :P, I'll update it as you said.

**Pull request merged**